aaudio/.google/packaging.yaml

```diff
@@ -0,0 +1,7 @@
+status:       PUBLISHED
+technologies: [Android, NDK]
+categories:   [NDK]
+languages:    [C++, Java]
```

Comment:
Does this have enough Java code to be in the Java category? Since it's NDK, maybe it should be listed under C++ only?